src/test/java/org/snakeyaml/engine/v2/tokens/TokenTest.java

```diff
@@ -30,14 +30,4 @@ void testToString() {
     Token token = new ScalarToken("a", true, Optional.empty(), Optional.empty());
     assertEquals("<scalar> plain=true style=: value=a", token.toString());
   }
-
-  @Test
-  void invalidDirectiveToken() {
-    List<String> list = Collections.singletonList("key");
-    YamlEngineException exception = assertThrows(YamlEngineException.class,
-        () -> new DirectiveToken(DirectiveToken.YAML_DIRECTIVE, Optional.of(list), Optional.empty(),
-            Optional.empty()));
-    assertEquals("Two strings/integers must be provided instead of 1", exception.getMessage());
-  }
 }
```

Comment:
The value of the `DirectiveToken` has been made typesafe, so this error is no longer possible